src/Input.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+
+import './style.css';
+
+class Inp extends React.Component {
+    render() {
+        let props = this.props;
```

Comment:
```suggestion
        let {type, value, placeholder} = this.props;
```
можно было еще так:)